exporters/zipkin/docs/api_diff_1.1.0_vs_1.0.0.txt

```diff
@@ -0,0 +1,4 @@
+Comparing source compatibility of  against 
+***  MODIFIED CLASS: PUBLIC FINAL io.opentelemetry.exporter.zipkin.ZipkinSpanExporter  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.util.logging.Logger baseLogger
```

Comment:
this is a bummer that I found while working on this. And, why I think we should make something like this check a part of our normal process.

Reply:
Totally agree with above statement.